build.gradle

```diff
@@ -37,6 +37,7 @@ dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlinVersion"
 
     compile "org.sert2521.sertain:core:$sertainVersion"
+    compile 'com.github.SouthEugeneRoboticsTeam:vision-plugin:ead1dfcdd4'
```

Comment:
Qué?

Reply:
Comes from a commit I rebased out of dev a few days ago. Aiden must have gotten it in that 30 second window, not bad! 😆